src/components/Header.jsx

```diff
@@ -65,16 +75,22 @@ export default () => (
         <NavMenus>
           <NavContainer>
             <NavMenuItem>
-              Intro
+              <NavItemLink href="#">Intro</NavItemLink>
+            </NavMenuItem>
+
+            <NavMenuItem>
+              <NavItemLink href="#HowItWorks">How It Works</NavItemLink>
             </NavMenuItem>
+
             <NavMenuItem>
-              How It Works
+              <NavItemLink href="#YourGain">Student Benefits</NavItemLink>
```

Comment:
I think it's better if we pass NavMenuItems through pages. That way the Header component can be used with the rest of our pages. Namely: "students" and "startups".

Reply:
Actually I think it's better if you remove all new additions to this component because alignment is also off.

It would be better done using Material UI instead of styled components. I will look into that once you're done removing all the new components